docs/tutorials/robusta-runner-metrics.rst

```diff
@@ -1,12 +1,20 @@
 Monitoring Robusta with Prometheus
 ======================================
 
-Metrics
-************
+Robusta Metrics Graphs
+**************************
+
+Robusta runner supports Prometheus and exposes performance and error metrics on the `/metrics` endpoint.
+
+After Prometheus discovers the ServiceMontior, Robusta runner app page will include the metrics graphs like the example below.
```

Comment:
Fix typo + streamline:

`After Prometheus discovers the ServiceMonitor, the robusta-runner page in the UI will automatically include graphs as shown below.`

Reply:
Thank you, fixed.